client/dashboard/data/index.ts

```diff
@@ -398,6 +398,13 @@ export const resetPhpMyAdminPassword = async ( siteIdOrSlug: string ): Promise<
 	} );
 };
 
+export const fetchPrimaryDataCenter = async ( siteIdOrSlug: string ): Promise< string > => {
```

Comment:
Looks like we can list the response value as follows and then we don't need `primaryDataCenter as keyof typeof dataCenterOptions` in other places 🤔

```
type DataCenter = 'bur' | 'dfw' | 'dca' | 'ams';

export const fetchPrimaryDataCenter = async ( siteIdOrSlug: string ): Promise< DataCenter >
```

Reply:
WDYT if we add the type in /client/data/data-center instead?

Reply:
Updated in e46f9d3